frontend/ui/src/components/SourceLogo/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {CSSProperties, SyntheticEvent} from 'react';
+import {ReactComponent as GoogleLogo} from 'assets/images/icons/google_avatar.svg';
```

Comment:
I think we should rename all of them so that they resemble as close as possible the file name as discussed with you and @AitorAlgorta last week.

In this case though, I would probably renamed the files :) as the components seem to have the right name?